src/constants.js

```diff
@@ -167,6 +168,29 @@ const constants = {
         selectedRows: PropTypes.array,
         invalidRows: PropTypes.array,
     }),
+
+    MAIN_CATEGORY_MAPPING: [
```

Comment:
Hm, why does this mapping need to be here?

As we read the keyword set `helsinki:topics` from the API, should that not be a sufficient list of the yso codes that we use? Better have a single source of truth so that if the keyword set in the API changes, the UI is not left behind?